client/src/Question.svelte

```diff
@@ -93,15 +93,15 @@
 		let when = new Date(q.when * 1000);
 		let dur = (now - when) / 1000;
 		if (dur > 24 * 60 * 60) {
-			return parseInt(dur / (24 * 60 * 60)) + "d ago";
+			return Math.trunc(dur / (24 * 60 * 60)) + "d ago";
```

Comment:
Unrelated to derived state mutation, but still looks like we are [encouraged](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#using_parseint_on_non-strings) to use `Math.trunc` for what we are trying to achieve here.